### PR TITLE
Added new devices and translated into german

### PR DESCRIPTION
**Pull request by mapulu**
Hopefully the app will continue soon and switch from alpha to beta to stable...

### CODE REVIEW

**Comment by mapulu** [comment]
Nope!

File: drivers/014G0159/driver.compose.json [hunk 1]
```diff
@@ -32,14 +33,16 @@
       "image": "{{driverAssetsPath}}/learnmode.svg",
       "instruction": {
         "en": "Press the button on top of the {{driverNameEn}} to start pairing.",
-        "nl": "Druk op de knop boven op de {{driverNameNl}} om het apparaat toe te voegen."
+        "nl": "Druk op de knop boven op de {{driverNameNl}} om het apparaat toe te voegen.",
+		"de": "Drücke die Taste oben auf dem Raum Thermostat"
```
Comment (KyleAthom) [inline drivers/014G0159/driver.compose.json]: Please keep the same indentation as the original file.

File: drivers/MT2646/driver.settings.compose.json [hunk 2]
```diff
@@ -97,21 +107,24 @@
...
+          "de": "Schalt-Status"
         }
       },
       {
         "id": "2",
         "label": {
           "en": "Night mode",
-          "nl": "Nacht modus"
+          "nl": "Nacht modus",
+          "nl": "Nacht Modus"
```
Comment (KyleAthom) [inline drivers/MT2646/driver.settings.compose.json]: Should be de I guess?

File: drivers/014G0159/driver.compose.json [hunk 3]
```diff
@@ -32,14 +33,16 @@
...
       }
     },
     "unlearnmode": {
       "image": "{{driverAssetsPath}}/learnmode.svg",
       "instruction": {
         "en": "Press the button on top of the {{driverNameEn}} to remove the device.",
-        "nl": "Druk op de knop boven op de {{driverNameNl}} om het apparaat te verwijderen."
+        "nl": "Druk op de knop boven op de {{driverNameNl}} om het apparaat te verwijderen.",
+		"de": "Drücke die Taste oben auf dem Raum Thermostat"
```
Comment (KyleAthom) [inline drivers/014G0159/driver.compose.json]: Raum Thermostat > {{driverNameN1}}

File: drivers/MT2646/driver.compose.json [hunk 4]
```diff
@@ -15,14 +16,16 @@
       "image": "{{driverAssetsPath}}/learnmode.svg",
       "instruction": {
         "en": "Put the {{driverNameEn}} in a socket or press the button on the front of the device three times to start inclusion.",
-        "nl": "Stop de {{driverNameNl}} in het stopcontact of druk drie keer op de knop aan de voorkant van het apparaat om het toevoegen te starten."
+        "nl": "Stop de {{driverNameNl}} in het stopcontact of druk drie keer op de knop aan de voorkant van het apparaat om het toevoegen te starten.",
+		"de": "Stecke das Gerät in eine Steckdose oder drücke die Taste auf der Vorderseite des Gerätes dreimal, um die inklusion zu starten."
```
Comment (KyleAthom) [inline drivers/MT2646/driver.compose.json]: Gerät > {{driverNameN1}}

File: drivers/MT2652/driver.flow.compose.json [hunk 5]
```diff
@@ -15,28 +16,32 @@
               "id": "1",
               "label": {
                 "en": "Top left",
-                "nl": "Links boven"
+                "nl": "Links boven",
+				"de": "Links oben"
```
Comment (KyleAthom) [inline drivers/MT2652/driver.flow.compose.json]: Check the indentation

File: drivers/MT2758/driver.settings.compose.json [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+			"de": "Notification Typ"
+        },
+        "value": "0",
+        "values": [
+            {
+                "id": "-128",
+                "label": {
+                    "en": "Notification Report",
+                    "nl": "Notification Report",
+					"de": "Notification Report"
```
Comment (KyleAthom) [inline drivers/MT2758/driver.settings.compose.json]: Indentation

File: drivers/MT2758/device.js [hunk 7]
```diff
@@ -35,9 +35,12 @@ class DevoloSiren extends ZwaveDevice {
                 size: 1,
 			}, 0)
 		    });		
+			this.DisableTamperSirenFlow = new Homey.FlowCardAction('MT2758disable_tamper').register().registerRunListener((args, state) => {
```
Comment (KyleAthom) [inline drivers/MT2758/device.js]: Please cancel the tamper alarm through a timeout. Check the Motion Sensor for an example.

**Comment by mapulu** [comment]
Will add new PRs soon

**Pull request closed without merging**